app/wama.web/Views/CheckIn/Waiver.cshtml

```diff
@@ -0,0 +1,80 @@
+            walls, exercise areas, equipment, participation in classes or events, and/or outdoor programs guided by or connected with the SDSU Wellness Center
+            Climbing Gym, whether the use is supervised or unsupervised. I, for myself and for my heirs, next of kin, assigns, personal representatives, and persons
+            upon whose behalf I am signing the “Sign-in Sheet,” hereby agree to and release, indemnify and hold harmless the State of South Dakota, South Dakota
+            Board of Regents, South Dakota State University, and the SDSU Wellness Center Climbing Gym, and their agents, employees, offices, and, volunteer
+            assistants, from any and all causes of action, claims for damages or demands whatsoever. THIS WAIVER AGREEMENT IS BINDING EVEN IF THE
+            RELEASED PERSON(S) OR ENTITY(IES) HAVE CAUSED OR CONTRIBUTED TO ANY DAMAGE OR INJURY THROUGH THEIR COLLECTIVE OR
+            INDIVIDUAL NEGLIGENCE.
+
+            I and/or person on whose behalf I am signing-in, voluntarily assume complete responsibility for risks and any injuries or damage which may occur as a result of
+            those risks even if the manner or type of injury or damage occurs in a manner that is not foreseeable at the time this Waiver Agreement is accepted. In
+            consideration of my use of the gym, its equipment, employees, volunteer assistants, independent contractors, I agree to and do release, indemnify and hold
+            harmless, the SDSU Wellness Center Climbing Gym, and any and all of their agents, servants and employees, from all liability, claims, demands and damages
+            and further promise not to commence any action or proceeding asserting same.
+
+            All climbers who are twelve (12) years of age or under must be directly supervised by an SDSU Wellness Center Climbing Gym approved adult or be a participant
+            in an SDSU Wellness Center Climbing Gym program. The “Sign-in Sheet” is being signed by the youth’s parent, legal guardian, or adult authorized to sign by the
+            youth’s parent or legal guardian. By signing the “Sign-in Sheet”, the adult acknowledges that they understand the terms of the Waiver Agreement and has the
+            authority to sign for the youth climber. The person signing the “Sign-in Sheet” understands and acknowledges that this Waiver Agreement is binding on the person
+            on whose behalf the “Sign-in Sheet” is signed, for their heirs, next of kin, assigns, and personal representatives.
+
+            BY SIGNING the SDSU Wellness Center Climbing Gym “SIGN-IN SHEET” I ACKNOWLEDGE THAT I HAVE READ AND AGREE TO THE TERMS OF THIS
+            WAIVER AGREEMENT. THERE ARE NO ORAL REPRESENTATIONS, STATEMENTS, OR INDUCEMENTS WHICH HAVE BEEN MADE THAT ALTER,
+            CHANGE OR MODIFY ANYTHING SET FORTH IN THIS WAIVER AGREEMENT.
+                        
+            <div class="form-group col-xs-12">
```

Comment:
Also, this area can be improved a bit.

**Adjusting the HTML a little bit**

![image](https://cloud.githubusercontent.com/assets/1667350/23154269/9c457e1a-f7d2-11e6-85bd-4b471586d1d1.png)

-----

**We can get something like this**

![image](https://cloud.githubusercontent.com/assets/1667350/23154259/8eaa1996-f7d2-11e6-9be5-2cb721dc0e03.png)